src/settings.ts

```diff
@@ -66,6 +66,46 @@ function getWeb3ConnectProviderOptions(network: string) {
           },
         },
       };
+    case "kovan":
+      return {
+        network: "kovan",
+        torus: {
+          package: Torus,
+          options: { network: "kovan" },
+        },
+        walletconnect: {
+          package: isMobileBrowser() ? null : WalletConnectProvider,
+          options: {
+            infuraId: "e0cdf3bfda9b468fa908aa6ab03d5ba2",
```

Comment:
does this id is the same for all networks ?

Reply:
I am assuming so, am planning to test the assumption.  We should of course test all of the wallets.

Reply:
why this infura id is in the code at all ? it should come as env var . it is private info , no ?

Reply:
I'm not sure all the wallets are going to work.

Reply:
why we have the infura token spread all over the code ?
it should get into env var . see #1569

Reply:
that is for another PR